examples/gocean/eg3/ocl_trans.py

```diff
@@ -48,18 +54,21 @@ def trans(psy):
     :rtype: :py:class:`psyclone.psyGen.PSy`
 
     '''
-    from psyclone.transformations import OCLTrans
+    ocl_trans = OCLTrans()
+    fold_trans = FoldConditionalReturnExpressionsTrans()
+    move_boundaries_trans = GOMoveIterationBoundariesInsideKernelTrans()
 
     # Get the Schedule associated with the first Invoke
     invoke = psy.invokes.invoke_list[0]
     sched = invoke.schedule
 
     # Transform the Schedule
-    cltrans = OCLTrans()
-    cltrans.apply(sched, options={"end_barrier": True})
+    ocl_trans.apply(sched, options={"end_barrier": True})
 
     # Provide kernel-specific OpenCL optimization options
     for kern in sched.kernels():
+        move_boundaries_trans.apply(kern)
```

Comment:
Added the comments, I slightly modified the README but not much because it doesn't go into details about the specific transformations and I think the overall descriptions still fit well.